data_steward/tools/clean_test_project.py

```diff
@@ -0,0 +1,75 @@
+# Python imports
+import logging
+import sys
+
+# Third party imports
+from googleapiclient.errors import HttpError
+
+# Project imports
+import app_identity
+import bq_utils
+
+logging.basicConfig(
+    stream=sys.stdout,
+    level=logging.DEBUG,
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+
+GITHUB_USERNAMES = []
+
+
+def get_circle_datasets(datasets_list):
+    """
+    Filters list of datasets with github usernames in them
+
+    :param datasets_list: list of dataset_ids
+    :return: list of dataset_ids with github usernames
+    """
+    circle_datasets = []
+    for dataset in datasets_list:
+        if any(gh_username in dataset for gh_username in GITHUB_USERNAMES):
+            circle_datasets.append(dataset)
+    return circle_datasets
+
+
+def delete_datasets(project_id, datasets_to_delete_list):
+    """
+    Deletes datasets using their dataset_ids
+
+    :param project_id: identifies the project
+    :param datasets_to_delete_list: list of dataset_ids to delete
+    :return: 
+    """
+    deleted, not_deleted = 0, 0
+    for dataset in datasets_to_delete_list:
+        try:
+            bq_utils.delete_dataset(project_id, dataset)
+            deleted += 1
+            logging.info('Deleted dataset %s' % dataset)
+        except HttpError:
+            logging.info('Could not delete dataset %s' % dataset)
```

Comment:
if you used `logging.exception(` here we could get the stack trace that goes with your error message.

Reply:
Yeah that makes sense.